### ISSUE TITLE
Enable acceptance of specific self-signed certificates

### ISSUE THREAD

**Issue by lenucksi** [nerzhul/ownCloud-SMS-App#172]
It would be nice to if I'd be prompted to accept a self-signed certificate by fingerprint instead of giving a generic error message that the connection was not possible.

**Comment by nerzhul** [comment]
Yeah it's an idea i get since a long time but i don't get time to do this atm and how to properly implement this. I'm just sure it's mandantory to have last HTTP code bug fixes

**Comment by s13m11t7** [comment]
I have just bought the app and cannot connect because of a self-signed certificate as well... Can't get a refund either.... The lack of support for a self-signed certificate needs to be clearly stated on the app page so we know not to buy and download it....
